boxoffice/mailclient.py

```diff
@@ -2,27 +2,33 @@
 
 from decimal import Decimal
 from flask import render_template
+from rq.decorators import job
```

Comment:
This should be from `from flask.ext.rq import job`. [See documentation](http://flask-rq.readthedocs.io/en/latest/).